packages/jellyfish-api-core/src/category/governance.ts

```diff
@@ -101,13 +101,15 @@ export class Governance {
    * @param {Object} options List proposal filter options
    * @param {ListProposalsType} [options.type=ListProposalsType.ALL] type of proposals
    * @param {ListProposalsStatus} [options.status=ListProposalsStatus.ALL] status of proposals
+   * @param {number} [options.cycle=0] payout cycle of proposals
```

Comment:
```suggestion
   * @param {number} [options.cycle=0] cycle: 0 (show all), cycle: N (show cycle N), cycle: -1 (show previous cycle)
```
^ suggestion to make the cycle param more informative

Reply:
updated